plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/CliOptions.java

```diff
@@ -42,7 +42,11 @@ public String getJenkinsWar() {
 
 
     public String[] getPlugins() {
-        return Arrays.copyOf(plugins, plugins.length);
+        if (plugins != null) {
+            return Arrays.copyOf(plugins, plugins.length);
+        }
+        return null;
```

Comment:
return empty array to avoid further issues? Maybe it also makes sense to change API to `List` or `Set` instead of the array